src/modules/sdk/objections/service.ts

```diff
@@ -27,6 +30,23 @@ const OBJECTIONS_API_SINGLE_ATTACHMENT_URL =
     (companyNumber: string, objectionId: string, attachmentId: string): string =>
       OBJECTIONS_API_URL(companyNumber) + `/${objectionId}/attachments/${attachmentId}`;
 
+/**
+ * Get the eligibility for the given company.
+ *
+ * @param {string} companyNumber the company number
+ * @param {string} token the bearer security token to use to call the api
+ *
+ * @returns {boolean} the eligibility for the given company
+ * @throws {ApiError}
+ */
+export const getEligibility = async (companyNumber: string, token: string): Promise<boolean> => {
```

Comment:
Should method name be 'eligible' perhaps, or 'isEligible'? Not sure of the Node standard on this.

I see that in PTF and Extensions we had an 'exists' method that returned a boolean...

Reply:
Yeah? Wasn't sure on the naming convention and figured I'd copy the other methods in the class

Reply:
how about getCompanyEligibility ?